package.json

```
@@ -0,0 +1,44 @@
+{
+  "name": "urban-brussels-api",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "tsc": "tsc",
+    "build": "tsc && netlify-lambda build functions",
+    "lint": "eslint . --ext .ts",
+    "start": "tsc && node -r esm ./build/app.js",
+    "start:server": "netlify-lambda serve functions",
+    "dev": "ts-node-dev -r esm --respawn --transpileOnly ./src/app.ts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```

Please switch to MIT license.
```suggestion
  "license": "MIT",
```